docker-compose.yml

```diff
@@ -105,6 +105,7 @@ services:
       - CHROME_URL=http://chrome:4444
       - DOCKER=true
       - CAPYBARA_PORT=5100
+      - CUCUMBER_PUBLISH_QUIET=true
```

Comment:
:heart: 

Want to add this to `web` too? The timeout keeps annoying me when debugging:
```
      # Disable rack-timeout
      - RACK_TIMEOUT_SERVICE_TIMEOUT=0
```